lab-gurpreet/readme.md

```diff
@@ -0,0 +1,6 @@
+as a developer
```

Comment:
So, it looks like you're still writing your readme as if you're making notes to yourself. That's not really what you want to do. The purpose of a readme is to provide OTHER developers with instructions on how to install, test, and use your application.

For example, right now, I wouldn't know how to run your cowsay CLI without going through your code. You should provide me with instructions on how to use your app.